src/attrib/parse.ts

```diff
@@ -63,9 +65,19 @@ export async function parseItemFromAttribFile(file: string, data: any, civ: civC
 
     const unique = parseUnique(ui_ext);
 
-    const costs = parseCosts(ebpExts?.cost_ext?.time_cost || data.upgrade_bag.time_cost, ebpExts?.population_ext?.personnel_pop);
+    let costs;
+    if (type === ITEM_TYPES.ABILITIES && file.startsWith("abilities"))
+      costs = parseCosts(data.ability_bag.cost_to_player, data.ability_bag.recharge_cost, 0 );
+    else if (type === ITEM_TYPES.ABILITIES && file.startsWith("info/buff_info"))
+      costs = {};
+    else
+      costs = parseCosts(ebpExts?.cost_ext?.time_cost?.cost || data.upgrade_bag?.time_cost?.cost, ebpExts?.cost_ext?.time_cost?.time_seconds || data.upgrade_bag?.time_cost?.time_seconds, ebpExts?.population_ext?.personnel_pop );
 
-    const icon = await useIcon(ui_ext.icon_name, type, id);
+    let icon;
+    if (type === ITEM_TYPES.ABILITIES && file.startsWith("info/buff_info")) 
+      icon = await useIcon(ui_ext.icon.slice(6), type, id);
```

Comment:
there's also icons tucked away in `ability_bag.ui_info.icon_name` for i.e. camel support.